### ISSUE TITLE
Practical Tips for your first start-up (Aarron White - CTO Boundless)

### ISSUE THREAD

**Issue by MarcoMorawec** [MarcoMorawec/SiB-Dev-Team#1]
**tl;dr: Do a ton of shit before you start building an actual product**

Use the [Business model canvas](http://www.businessmodelgeneration.com/canvas) to write down what you want to do - screw a full business plan.

Use the [validation board from the lean startup machine](http://tech.co/wp-content/uploads/2012/10/Lean-Startup-Machine-validation-board.png) and show investors how you went through all the risk assumptions and how you in-validated them. This will impress them even without an actual product.

Build outside-in and ask: 
    \* Where will people find me
    \* what add will pique their interest
    \* what value do I need to sell them on
    \* Can they that value fast as possible
    \* Do I maintain a good relationship
    \* Have I earned their dollar?

Build a landing page and integrate a chat box to talk to people on the site and ask them more questions

Follow-up with every single sign-up that you get: "Hey I'm Marco and I'm stoked you signed up, -> then ask them your one riskiest assumptions and see if it's true".

**Picking a co-founder**

look for technical talent 10 months in advance and don't do a rushed thing at the very end. Feed them the idea, bit by bit and develop a relationship and then ask them to come on board.

Show them how you de-risked one risk after another and get them slowly excited to the idea.

**Employees**

The first 10 people you're hire set the tone of how fast you work and what other talent you attract later on.

Boundless looked for 4 types to have in a new hire:
    \* the beast from x-mean (somebody who attacks problems
    \* Indiana Jones (somebody you can take risks and likes to start something)
    \* the architect from the matrix (the person who has the whole pictures of your biz in their head and knows their field really well and knows all the details in their field)
    \* Dos Equies guy (if the person was the last person in the office, would you go have a beer with him)

have final round interviewees present a 15 min talk about something that they're interested in. Why? If they're good communicators they can create/push change within your team…if they can't communicate well, they won't enable any change.

**Investors**

get the smart money that can help you guide.

**Outline of a good pitch**

*Cover
*Hook
*Team
*Problem
*Opportunity
*Solution
*Proof -> include all the risk assumption that you invalidated to decide building this startup in the first place
*Unfair advantage (what's your companies superpower, why can you execute better than everyone else) -> we understand consumer behavior, we understand crowd sourcing, etc….and there is no way our competitor could do all this
*Market Size -> you need to have this
*Roadmap (with value creation milestones - VCM): what are you doing, and what are you going to reach when, what are the specific things you are going to do

> "Look we told our investors that we could do x,y,z in 1 year but we've done that it works in just 6 months" -> we're executing really well
> *THE ASK -> tie to VCM ("how do allocate the money that you might get?")

**Fundraise quickly**
The longer you stay in fundraising mode, the worst off you are. You look like a horse that nobody wants to bet on it otherwise.
Cancel all consulting gigs and do it full time and try to impress every single person

**Creating scarcity (during your fundraise)**
create artificial scarcity: "we're going to the 4th popular seed funds in Boston, and we told each of them, we want to pitch to you and then open it up to a larger crowd -> this create urgency and get's you meetings to pitch fast.

**Value creation milestones**
the company should not grow linearly…what shifts your business upward fast is what you put in your milestones.
e.g. "the most popular app has 100k downloads, we got 50k in one month, now we're in striking distance"

**One big success metric of startups:**
"What's your rate of learning?" -> focus on getting your iteration times as tight as possible
